modules/internal/DefaultAssociative.chpl

```diff
@@ -430,14 +430,14 @@ module DefaultAssociative {
       }
     }
   
-    iter dsiSorted() {
+    iter dsiSorted(comparator) {
```

Comment:
It's internal, so I don't think it matters. The optional arg is handled in ChapelArray.